status/status.go

```diff
@@ -286,11 +286,17 @@ func (s *Status) generateTestReport(ctx context.Context, detail intgteststat.Int
 		return nil, fmt.Errorf("failed to generate summary message: %w", err)
 	}
 
+	fullName := fmt.Sprintf("%s / %s", NamePrefix, detail.ScenarioName)
+	if snapshot.Labels[gitops.SnapshotComponentLabel] != "" {
+		fullName = fmt.Sprintf("%s / %s", fullName, snapshot.Labels[gitops.SnapshotComponentLabel])
```

Comment:
Why are we swapping order?

`app > component > test` from the most generic to the most specific looks more natural

Reply:
When testing it out, it felt more natural and useful to have the integrationTestScenario as the first item. This was especially important in gitLab since the commitStatus sometimes gets truncated, and in most cases the biggest differentiator between test runs is the scenario.
Also since we can expect some Snapshot to not have the component, having it as the last item makes it more consistent.

Reply:
Snapshot without component won't be reported into github, as it won't be a PR processed by build-service. It could be max an accident when user copy paste wrong data or uses another integration.

But if it's for better gitlab readability, then ok